borg/helpers.py

```diff
@@ -989,6 +989,8 @@ def __init__(self, total, step=5, start=0, same_line=False, msg="%3.0f%%", file=
         self.total = total
         self.trigger_at = start  # output next percentage value when reaching (at least) this
         self.step = step
+        if file is None:
+            file = sys.stderr
         self.file = file
```

Comment:
True, although i chose this to match the style used in 7abcaa0d9d34b9629b88b4e9ddf62ca0f76ade6b.

As a bonus, this (longer) form reads clearer, since it's almost an english sentence - "If file is none, file shall be stderr".

If @ThomasWaldmann doesn't object, I'd like to leave it like it is now (mainly for the symmetry reason to `archive.py`)